pages/index.tsx

```diff
@@ -45,7 +45,7 @@ const Home: NextPage = () => {
   return (
     <>
       <Head>
-        <title>Codú - The Web Developer Community</title>
+        <title>Codú -  Web Developer Community</title>
```

Comment:
Could we revert this for the moment?

Reply:
This was by mistake, i think i fixed this in the last commit.